dev/com.ibm.ws.management.security/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -21,5 +21,13 @@ role.administrator.desc=A collection of users and/or groups assigned the server
 role.user=User name
 role.user.desc=User assigned a role.
 
+role.userAccessId=User access ID
+role.userAccessId.desc=A user access ID in the general form user:realmName/userUniqueId. A value will be generated if one is not specified.
```

Comment:
We try to keep sentences in present tense. How about "A value **is** generated if one is not specified."?

Reply:
fixed